### ISSUE TITLE
Menu Scrolling with Mouse

### ISSUE THREAD

**Issue by George-R-Young** [xbmc/xbmc#15570]
### Describe the bug
Here is a clear and concise description of what the problem is:

In the Kodi 18.1 menu, if I move the mouse down, the menu scrolls up to show the lower items. However, if I move the mouse up, the menu does not scroll down. As a result, upper items on the menu become unavailable.
## Expected Behavior
Here is a clear and concise description of what was expected to happen:

I think if I move the mouse up, the menu should scroll down.

## Actual Behavior

With mouse motion, menu scrolls up but not down.

### To Reproduce

1. see Main Menu
2. see Add-Ons Menu
3.

### Debuglog

The debuglog can be found here:

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [x] Android

 - Operating system version/name: Android 7.1.2 Nougat
 - Kodi version: 18.1 Leia

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by George-R-Young** [comment]
Debugging log: https://paste.kodi.tv/oyesavamum

**Comment by DaVukovic** [comment]
You hvae banned add-ons installed. Remove then and provide a new log

**Comment by George-R-Young** [comment]
Alrighty, I uninstalled Kodi, the installed fresh from the Play Store. The only add-ons are Netflix and the Kodi Log Uploader, neither of which, I believe, are banned.
Here's the fresh Debugging Log: https://paste.kodi.tv/sagifiteji
FYI, the menu scrolling problem remained.

**Comment by DaVukovic** [comment]
Works for me as expected at the Kodi library. Could be a problem with the netflix add-on itself?!

If I use my mouse on Ubuntu, I see log messages like:

```
2019-02-25 06:18:10.020 T:139676929136768   DEBUG: ------ Window Init (Pointer.xml) ------
2019-02-25 06:18:16.701 T:139676929136768   DEBUG: ------ Window Deinit (Pointer.xml) ------
2019-02-25 06:18:16.835 T:139676929136768   DEBUG: ------ Window Init (Pointer.xml) ------
```

Which I don't see in your log. Are you using a real/normal PC mouse? If yes, does the scroll wheel works as expected (as it works fine for me)? 

If you don't use a PC mouse, what are you using instead?

**Comment by George-R-Young** [comment]
BTW, this is Android, not linux.

I use an airmouse. I'll try with a conventional mouse instead and generate another debug file.

**Comment by George-R-Young** [comment]
Ok, I tried using a conventional PC mouse and here's the debug log:
https://paste.kodi.tv/wunotezene

The problem is unchanged.

Let me try to explain the problem in more detail.

When you fire up kodi, the main menu looks like:
Movies
TV Shows
Music
Music Videos
TV
Radio
Games
Add-ons
Pictures

If you hover the mouse pointer over the menu and move it down, the menu
moves up and looks like:
TV
Radio
Games
Add-ons
Pictures
Videos
Favourites
Weather

Three more entries on the bottom, four fewer on the top.

However, when you hover the mouse pointer over the menu and move it up,
the menu does not move down. The top four entries in the menu:
Movies
TV Shows
Music
Music Videos

are not visible or selectable.

I think it's a bug if the menu only scrolls up, not down, with mouse pointer motion.

**Comment by DaVukovic** [comment]
True, I can reproduce that on Linux as well. But only for the main/home menu items.

**Comment by George-R-Young** [comment]
I have the same problem in the Add-ons menu. However, the Add-ons/Kodi Add-on Repository menu works fine, scrolling in both directions based on mouse motion.
